nix/default.nix

```diff
@@ -1,13 +1,20 @@
 { system ? builtins.currentSystem
 , nixpkgs ? (import ./sources.nix { inherit system; }).nixpkgs
+, overlays ? [
```

Comment:
why is all this a toplevel arg? It doesn't make a lot of sense, we cannot pass overlays, pkgs or pkgsAarch64.

Can you move it down?

Reply:
brain empty, not a single thought inside.